src/plone/restapi/services/vocabularies/get.py

```diff
@@ -24,7 +36,24 @@ def _error(self, status, type, message):
         self.request.response.setStatus(status)
         return {"error": {"type": type, "message": message}}
 
+    def _has_permission_to_access_vocabulary(self, vocabulary_name):
+        """Check if user is authorized to access built-in vocabulary
+
+        default permission for all vocabularies, built-in and others, was
+          <permission
+            id="plone.restapi.vocabularies"
+            title="plone.restapi: Access Plone vocabularies"
+            />
+        """
+        if vocabulary_name in PERMISSIONS:
```

Comment:
We need to always check the `DEFAULT_PERMISSION` in order to get the same defined behavior as here:
https://github.com/plone/plone.app.content/blob/7a1bd2b60136b1fd380d4fa04bd052498548f272/plone/app/content/browser/vocabulary.py#L304-L324

Reply:
I skip the field permission check as this is not the responisbility of the vocabulary, but that of the schema and its consumer.

Reply:
I agree here.

Anyway,  `if vocabulary_name in PERMISSION` is wrong/ superfluous. `DEFAULT_PERMISSION` must always be checked to match the logic in the above referenced current code.

Reply:
One could argue, if it needs check if it is different from the permission the view is registered with. 

Thinking further, the endpoint can be registered with `zope.Public` in ZCML. The default goes with `zope.View` as is. Just, if anyone in an add-on needs it, a vocabulary could be registered with `zope.Public`. This way we do not disallow this (probably rare) use case by design.

Reply:
I understand that DEFAULT_PERMISSION needs to be checked if the same behavior as VocabularyView.get_vocabulary of https://github.com/plone/plone.app.content/blob/7a1bd2b60136b1fd380d4fa04bd052498548f272/plone/app/content/browser/vocabulary.py#L304-L324 should be followed in vocabulary endpoint.
It's not in scope of this PR to think about why a vocabulary is a utility which cannot be registered with a permission. Which means that the DEFAULT_PERMISSION is supposed to fit for custom vocabularies.

I will take just the one step to follow the classic behavior in vocabulary endpoint and check DEFAULT_PERMISSION for all vocabularies.